### PR TITLE
refactor: progress full token

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   -        |
| 🇨🇳 Chinese |    -       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/progress/style/index.tsx [hunk 1]
```diff
@@ -98,10 +97,10 @@ const genBaseStyle: GenerateStyle<ProgressToken> = (token: ProgressToken) => {
...
       [`${progressCls}-text`]: {
         display: 'inline-block',
-        width: '2em', // FIXME: hardcode in v4
-        marginInlineStart: token.marginXS, // FIXME: hard code in v4
-        color: token.infoTextColor,
-        fontSize: token.textFontSize,
+        width: '2em',
+        marginInlineStart: token.marginXS,
+        color: token.progressInfoTextColor,
+        fontSize: token.progressTextFontSize,
```
Comment (zombieJ) [inline components/progress/style/index.tsx:103]: 上面都 resetComponent 了这边还需要再设置一下 fontSize 么？

File: components/progress/style/index.tsx [hunk 2]
```diff
@@ -85,8 +84,8 @@ const genBaseStyle: GenerateStyle<ProgressToken> = (token: ProgressToken) => {
       [`&${progressCls}-success-bg, ${progressCls}-bg`]: {
         position: 'relative',
         backgroundColor: token.blue,
```
Comment (zombieJ) [inline components/progress/style/index.tsx:86]: 感觉是 colorInfo?

File: components/progress/style/index.tsx (hunk 1 above)
Comment (zombieJ) [reply components/progress/style/index.tsx:103]: 另一个 progressCircleTextFontSize 也是同样的疑问

File: components/progress/style/index.tsx [hunk 3]
```diff
@@ -117,10 +116,10 @@ const genBaseStyle: GenerateStyle<ProgressToken> = (token: ProgressToken) => {
           position: 'absolute',
           inset: 0,
           background: token.colorBgComponent,
-          borderRadius: 10, // FIXME: hard code in v4
+          borderRadius: token.progressLineRadius,
           opacity: 0,
           animationName: antProgressActive,
-          animationDuration: '2.4s', // FIXME: hard code in v4
+          animationDuration: '2.4s', // FIXME: hard code
```
Comment (zombieJ) [inline components/progress/style/index.tsx:122]: 试试 durationSlow 可否代替掉？

File: components/progress/style/index.tsx (hunk 1 above)
Comment (MadCcc) [reply components/progress/style/index.tsx:103]: 去掉看看

File: components/progress/style/index.tsx (hunk 3 above)
Comment (MadCcc) [reply components/progress/style/index.tsx:122]: durationSlow 是 0.3s，差太多了。
其他地方也有很多长动画，比较零碎

**Review by zombieJ — APPROVED** [review]

**Pull request merged**